tidb-cloud/limited-sql-features.md

```diff
@@ -97,8 +97,6 @@ TiDB Cloud works with almost all workloads that TiDB supports, but there are som
 | `information_schema` | `TIDB_HOT_REGIONS` | Not supported [^4] | Not supported [^4] |
 | `information_schema` | `TIDB_HOT_REGIONS_HISTORY` | Supported | Not supported [^1] |
 | `information_schema` | `TIDB_SERVERS_INFO` | Supported | Not supported [^1] |
-| `information_schema` | `TIFLASH_SEGMENTS` | Supported | Not supported [^1] |
-| `information_schema` | `TIFLASH_TABLES` | Supported | Not supported [^1] |
```

Comment:
@breezewish Please invite a tech reviewer and remove the following notes:

https://github.com/pingcap/docs/blob/c4cb48307cc7a9b7580aa01445a4d848a0c0ac81/information-schema/information-schema-tiflash-tables.md?plain=1#L11-L14

https://github.com/pingcap/docs/blob/c4cb48307cc7a9b7580aa01445a4d848a0c0ac81/information-schema/information-schema-tiflash-segments.md?plain=1#L11-L14

Reply:
Thank you! Nice catch.